suite/tests/api/ir_x86.c

```diff
@@ -105,6 +105,7 @@ static byte buf[8192];
 #define IMMARG(sz) opnd_create_immed_int(37, sz)
 #define TGTARG opnd_create_instr(instrlist_last(ilist))
 #define REGARG(reg) opnd_create_reg(REG_##reg)
+#define DR_REGARG(reg) opnd_create_reg(DR_REG_##reg)
```

Comment:
I would say just change REGARG: no need for a separate thing.

Reply:
Done.